### DISCUSSION TITLE
What version of capacitor is required for this plug-in to work?

### DISCUSSION THREAD

**Discussion by ericallenkrueger** [capawesome-team/capacitor-mlkit#37] [Q&A] [answered]
What is the minimum version of capacitor for this plugin to work?  Thanks!
Eric

**Comment by robingenz** [comment, marked as answer]
You can use Capacitor 4 with plugin version 0.0.3 and Capacitor 5 with plugin version 5.0.0 (see [CHANGELOG.md](https://github.com/capawesome-team/capacitor-mlkit/blob/main/packages/barcode-scanning/CHANGELOG.md)).

Please note that we only provide updates for the latest Capacitor version.
